src/hipscat_import/file_readers.py

```diff
@@ -122,14 +122,40 @@ def provenance_info(self) -> dict:
 
 
 class FitsReader(InputReader):
-    """Simple, non-chunked FITS file reader."""
+    """Chunked FITS file reader."""
+
+    def __init__(
+        self,
+        chunksize=500_000,
+        column_names=None,
+        skip_column_names=None,
+    ):
+        self.chunksize = chunksize
+        self.column_names = column_names
+        self.skip_column_names = skip_column_names
 
     def read(self, input_file):
-        """Read the whole fits file and return"""
-        yield Table.read(input_file, format="fits").to_pandas()
+        """Read chunks of rows in a fits file"""
+        table = Table.read(input_file, memmap=True)
+        if self.column_names:
+            table.keep_columns(self.column_names)
+        elif self.skip_column_names:
+            table.remove_columns(self.skip_column_names)
```

Comment:
Can you add a line in the docstring saying how this works? (i.e. how skip_column_names is ignored if both are defined)